.rubocop.yml

```diff
@@ -307,3 +307,6 @@ GraphQL/ObjectDescription:
 
 GraphQL/ExtractType:
   Enabled: false
+
+GraphQL/ExtractInputType:
```

Comment:
This one makes sense I think. We should not ignore it.

Reply:
I agree, but sometimes 😓 

Here is an example of rubocop warnings on this PR:
```
argument :id, ID, required: true
argument :name, String, required: true
argument :value, Float, required: true
```

All these arguments have `Consider moving arguments to a new input type (convention:GraphQL/ExtractInputType)` and I'm not sure about this... 😅